### PR TITLE
[GraphQL/Docs] Comments for SuiSystemStateSummary et al.

### PR DESCRIPTION
**Pull request by amnn**
## Description

Also:

- `SystemParameters`
- `StakeSubsidy`
- `ValidatorSet` (although this is missing some fields)
- `SafeMode`
- `StorageFund`

## Test Plan

:eyes:

### CODE REVIEW

File: crates/sui-graphql-rpc/schema/current_progress_schema.graphql [hunk 1]
```diff
@@ -792,43 +803,145 @@ enum StakeStatus {
...
+Parameters related to subsiding staking rewards
+"""
 type StakeSubsidy {
+	"""
+	SUI set aside for stake subsidies -- reduces over time as stake subsidies are paid out over
+	time.
+	"""
 	balance: BigInt
+	"""
+	Number of times stake subsidies have been distributed.
```
Comment (wlmyng) [inline crates/sui-graphql-rpc/schema/current_progress_schema.graphql:816]: suggestion: could also mention how often this distribution occurs? iirc it's done per epoch

File: crates/sui-graphql-rpc/src/types/validator_set.rs [hunk 2]
```diff
@@ -5,11 +5,19 @@ use super::big_int::BigInt;
 use super::validator::Validator;
 use async_graphql::*;
 
+/// Representation of `0x3::validator_set::ValidatorSet`.
 #[derive(Clone, Debug, PartialEq, Eq, SimpleObject, Default)]
 pub(crate) struct ValidatorSet {
+    /// Total amount of stake for all active validators at the beginning of the epoch.
```
Comment (wlmyng) [inline crates/sui-graphql-rpc/src/types/validator_set.rs:11]: it doesnt seem like these comments are appearing in the schema?

**Review by stefan-mysten — COMMENTED** [review]
I also did the `ValidatorSet` but let's keep yours, they look better. I'll remove mine from the other PR to avoid conflicts.

File: crates/sui-graphql-rpc/src/types/validator_set.rs (hunk 2 above)
Comment (amnn) [reply crates/sui-graphql-rpc/src/types/validator_set.rs:11]: Ah, that was a bad rebase on my part, I think -- thanks for spotting.

**Review by wlmyng — APPROVED** [review]

**Pull request merged**